app/public/src/game/components/battle-manager.ts

```diff
@@ -541,6 +541,20 @@ export default class BattleManager {
                 this.flip
               )
             }
+          } else if (field == "powerWhipCount") {
```

Comment:
a custom counter for powerWhip doesn't seem necessary, you can use `broadcast(Transfer.ABILITY)` to display anim sprite on several targets. Look at TorchSongStrategy for example